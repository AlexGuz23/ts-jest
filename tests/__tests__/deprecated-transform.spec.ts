import runJest from '../__helpers__/runJest';

describe('Deprecated Jest transform value', () => {
  it('should log the depration message', () => {
    const result = runJest('../deprecated-transform', ['--no-cache']);

    const output = result.stderr;

    // get only the line with deprecation message (we dont want the snapshot to contain any timing or test name)
    const msg = output.split('\n').find(line => /deprecated/i.test(line));

    expect(msg).toMatchSnapshot();
  });
});
