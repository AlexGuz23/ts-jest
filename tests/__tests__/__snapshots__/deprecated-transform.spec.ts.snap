// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Deprecated Jest transform value should log the depration message 1`] = `"[ts-jest][DEPRECATED] - replace any occurrences to \\"ts-jest/dist/preprocessor.js\\" or  \\"<rootDir>/node_modules/ts-jest/preprocessor.js\\" in the 'transform' section of your Jest config with just \\"ts-jest\\"."`;
